### ISSUE TITLE
these files conflict

### ISSUE THREAD

**Issue by JaydenFish** [qlibs/qlibs#1]
```C++
#include <qlibs/jmp>
#include <qlibs/mem>
#include <qlibs/mp>
#include <qlibs/mph>    
#include <qlibs/reflect>
#include <qlibs/swar>
#include <qlibs/ut>
```
Error:
```
In file included from include/qlibs/mem:256:
/usr/include/sys/mman.h:81:12: error: exception specification in declaration does not match previous declaration
   81 | extern int mprotect (void *__addr, size_t __len, int __prot) __THROW;
      |            ^
include/qlibs/jmp:661:16: note: previous declaration is here
  661 | extern "C" int mprotect(void*, __SIZE_TYPE__, int);
      |                ^
1 error generated.
```

**Comment by krzysztof-jusiak** [comment]
Thanks @JaydenFish, can you please share your env (system, compiler, cxxflags, libc, libc++). It's likely  because of __THROW but can't repro on godbolt and locally so I'm defo missing something. thanks.

**Comment by krzysztof-jusiak** [comment]
Thanks, fixed now with jmp-5.0.1 (to be released tonight) - https://godbolt.org/z/o18cxx7YT
